config/locales/en.yml

```diff
@@ -1141,7 +1141,12 @@ en:
       bank_accounts:
         label: Money in bank accounts
         hints:
-          - For example, money in current or savings accounts, ISAs, or child accounts your client is a signatory on. Do not include any government Cost of Living Payments owed to your client.
+          - For example, money in current or savings accounts, ISAs, or child accounts your client is a signatory on.
+          - Do not include any government Cost of Living Payments owed to your client.
+          - Enter 0 if this does not apply.
+        with_partner_hints:
+          - For example, money in current or savings accounts, ISAs, or child accounts your client is a signatory on. If the client and partner share any joint bank accounts, include these.
```

Comment:
`If the client and partner share any joint bank accounts, include these.` This should be on line 1144 as part of the client hint